src/__tests__/buildConfig-test.js

```diff
@@ -228,4 +230,143 @@ test("buildConfig extends rules by replacing the prior rule config completely, n
   }).catch(logError)
 })
 
+test("buildConfig correctly handles `ignoreFiles`", t => {
+  const cwd = process.cwd()
+  process.chdir(__dirname)
+
+  buildConfig({
+    ignoreFiles: [
+      "/ignore.css",
+      "ignore-too.css",
+    ],
+    rules: {
+      "at-rule-empty-line-before": [ "always", {
+        expect: ["blockless-group"],
+      } ],
+    },
+  }).then(({ config }) => {
+    t.deepEqual(config, {
+      ignoreFiles: [
+        "/ignore.css",
+        path.join(__dirname, "ignore-too.css"),
+      ],
+      rules: {
+        "at-rule-empty-line-before": [ "always", {
+          expect: ["blockless-group"],
+        } ],
+      },
+    })
+    process.chdir(cwd)
+    t.end()
+  }).catch(logError)
+})
+
+test("buildConfig have `ignorePatterns` if `ignorePath` found", t => {
+  buildConfig({
+    ignorePath: path.join(__dirname, "./fixtures/ignore.txt"),
+    rules: {
+      "at-rule-empty-line-before": [ "always", {
+        expect: ["blockless-group"],
+      } ],
+    },
+  }).then(({ config }) => {
+    t.deepEqual(config, {
+      ignorePath: path.join(__dirname, "./fixtures/ignore.txt"),
+      ignorePatterns: "# Ignore file patterns are always relative to process.cwd()\n\nsrc/__tests__/fixtures/empty-block.css\n",
+      rules: {
+        "at-rule-empty-line-before": [ "always", {
+          expect: ["blockless-group"],
+        } ],
+      },
+    })
+    t.end()
+  })
+    .catch(logError)
+})
+
+test("buildConfig have not `ignorePatterns` if `ignorePath` not found", t => {
+  buildConfig({
+    ignorePath: "/nonexistent.js",
+    rules: {
+      "at-rule-empty-line-before": [ "always", {
+        expect: ["blockless-group"],
+      } ],
+    },
+  }).then(({ config }) => {
+    t.deepEqual(config, {
+      ignorePath: "/nonexistent.js",
+      rules: {
+        "at-rule-empty-line-before": [ "always", {
+          expect: ["blockless-group"],
+        } ],
+      },
+    })
+    t.end()
+  })
+    .catch(logError)
+})
+
+test("buildConfig throw error if we have file system problem with `ignorePath`", t => {
+  let planned = 0
+
+  const fsStub = sinon.stub(fs, "readFile")
+  fsStub.yields(new Error("Some file system error"))
+
+  buildConfig({
+    ignorePath: "/nonexistent-ignore.js",
+    rules: {
+      "at-rule-empty-line-before": [ "always", {
+        expect: ["blockless-group"],
+      } ],
+    },
+  }).catch(err => {
+    t.equal(err.message, "Some file system error")
+    fsStub.restore()
+  })
+  planned += 1
+
+  t.plan(planned)
+})
+
+test("buildConfig loads processors", t => {
+  buildConfig({
+    processors: [path.join(__dirname, "./fixtures/processor-html.js")],
```

Comment:
Can you make this a relative path so we confirm that `buildConfig` converts it to absolute?